devel/irrlichtmt/Portfile

```diff
@@ -18,4 +19,20 @@ checksums           rmd160  4da410c24f67e8c0ebd9880308b8d796d2f86676 \
 
 depends_lib         path:include/turbojpeg.h:libjpeg-turbo \
                     port:libpng \
-                    port:zlib
+                    port:mesa
```

Comment:
If we are only requiring mesa on specific versions of macOS, it should be gated with an `if`.

Given that we only need a header file, it should be in `depends_build`, not `depends_lib`.

Reply:
How to learn of the apparent findings of yours by help of the build checks please?

Reply:
You mean the GitHub Actions check or the PR template?

For continuous integration tests, it's hard to automate detection of unnecessary dependencies because dependencies could be introduced by `dlopen()` and scripting, which are hard to detect.

Regarding the PR template, we need to find a balance between overwhelming contributors and the bar of PR quality. I would recommended new contributors to read the [MacPorts Guide](https://guide.macports.org/) and [Portfile Recipes](https://trac.macports.org/wiki/PortfileRecipes), but it should not be a strict requirement for opening pull requests.